python/lsst/ap/verify/dataset.py

```diff
@@ -142,7 +142,7 @@ def calibLocation(self):
     def defectLocation(self):
         """The directory containing defect files (`str`, read-only).
         """
-        return self.calibLocation
+        return os.path.join(getPackageDir('obs_decam_data'), 'decam', 'defects')
```

Comment:
Please do not hard-code dependencies on specific instruments into `ap_verify`. If defects are no longer part of the dataset format, then removing the `defectLocation` member would be more appropriate.

Reply:
Removing defectLocation required more surgery than I had hoped to do.  Do you think you could have a look and suggest how best to do that?

Reply:
It looks like, outside of test code, the only place where `defectLocation` gets used is in your call to `_doIngestDefects`. So it might make more sense to make that a config field in `DatasetIngestConfig`, which will at least be consistent with how the inputs to `IngestCalibsTask` are handled.

The defect target could be configured in the dataset-specific config files, but it could also be put in an obs package config override file (already supported as `datasetIngest.py`). I think you need to modify those files anyway because you removed config fields.

Reply:
@kfindeisen I've implemented your suggestions.  I've updated `obs_test` and `obs_decam` though on a different ticket since they have to be merged together anyway: tickets/DM-18739 and tickets/DM-19730 respectively.  I didn't see anything to change in `obs_subaru`.  I hope this addresses your concerns.

@mrawls if you are happy with things as they stand, will you approve this please, or let me know what else I need to do?